db/schema.rb

```diff
@@ -10,13 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180820145623) do
+ActiveRecord::Schema.define(version: 20180821173444) do
 
   create_table "users", force: :cascade do |t|
```

Comment:
how was this table already created when the migration to add the table is in this PR?

Reply:
Not sure, throughout the entire tutorial, I've encountered db errors when trying to run `rails test`. At times these were resolved with `rails db:rollback` , but at one point I needed to reset the whole thing. I thought it might have something to do with my system, since a lot of the stackoverflow questions regarding this issue were from folks using Windows.

Not an answer, I know, but the whole flow about db migration is something that still makes little sense to me and it's on my list of things to discuss tomorrow.

Reply:
sounds good, we'll cover this tom.